aws/load_balancer/event_rule.yaml

```diff
@@ -0,0 +1,169 @@
+LoadBalancerCreateEventRule:
+  Type: AWS::Events::Rule
+  Properties:
+    EventBusName: default
+    EventPattern:
+      source:
+        - aws.elasticloadbalancing
+      detail-type:
+        - AWS API Call via CloudTrail
+      detail:
+        eventSource:
+          - elasticloadbalancing.amazonaws.com
+        eventName:
+          - prefix: CreateLoadBalancer
+    Name: port-aws-exporter-sync-elb-create-trails
+    State: ENABLED
+    Targets:
+      - Id: PortAWSExporterEventsQueue
+        Arn:
+          Fn::ImportValue:
+            Fn::Sub: ${PortAWSExporterStackName}-EventsQueueARN
+        InputTransformer:
+          InputPathsMap:
+            awsRegion: $.detail.awsRegion
+            eventName: $.detail.eventName
+            requestElbName: $.detail.requestParameters.name
+          InputTemplate: |-
+            {
+              "resource_type": "AWS::ElasticLoadBalancingV2::LoadBalancer",
+              "region": "\"<awsRegion>\"",
+              "identifier": "\"<requestElbName>\"",
+              "action": "if \"<eventName>\" | test(\"DeleteLoadBalancer[^a-zA-Z]*$\") then \"delete\" else \"upsert\" end"
+            }
+LoadBalancerUpdateEventRule:
```

Comment:
I think splitting the Cloudtrail events to different rules makes the file long and hard to read, 
Thinking about combining the event rules here to just one, what do you think?